Add 'data:' URL scheme compat data

Elchi3 opened the pull request:
If I see it correctly, this page is the last one in the MDN HTTP area that uses the old style (static) compat table :-)

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

wbamberg commented on http/data-url.json:
```diff
@@ -0,0 +1,63 @@
+              },
+              "edge": {
+                "version_added": "12",
+                "notes": ["IE9 and later, as well as Edge, supports data URLs in CSS and JS files, but not in HTML files, with a max size of 4GB."]
```
I find this note hard to read :).

First, I have tried to avoid referring to browsers in the note itself, because it ends up reading weirdly (why is the Edge data talking about IE?).

Second, the complex run-on sentence makes it ambiguous and hard to parse. It could be read as: `data` is supported in CSS, JS, HTML. In CSS and JS the max size is 4GB.

It would be better written as: "data URLs can only be used in CSS and JS files. The maximum size supported is 4GB."

wbamberg commented on http/data-url.json:
```diff
@@ -0,0 +1,63 @@
+              "ie": {
+                "version_added": "8",
+                "notes": ["IE8 only supports data URLs in CSS files, with a max size of 32kB",
+                          "IE9 and later, as well as Edge, supports data URLs in CSS and JS files, but not in HTML files, with a max size of 4GB."]
```
These two notes make me wonder if it's worth using subfeatures here for "CSS files"/"JS files", "HTML files".

Elchi3 commented:
Thanks @wbamberg! I just copied the static table without thinking too much. But you are right, lets migrate this properly. I've split this into subfeatures and added notes about the size limit to the Edge and IE versions.
